consensus/bor/bor.go

```diff
@@ -1256,6 +1230,27 @@ func applyMessage(
 	return nil
 }
 
+// fetch message
+func (c *Bor) fetchMessage(msg callmsg, header *types.Header) ([]byte, error) {
+	state, err := c.bc.StateAt(header.Root)
```

Comment:
What happens when the state is null. We had this issue and that's why we were using `header.parentHash` for `GetCurrentSpan` before. 

@ferranbt can you look into this once?